docs/kpod-login.1.md

```diff
@@ -0,0 +1,67 @@
+% kpod(1) kpod-login - Simple tool to login to a registry server
+% Urvashi Mohnani
+# kpod-login "1" "August 2017" "kpod"
+
+## NAME
+kpod-login - Login to a container registry
+
+## SYNOPSIS
+**kpod login**
+**REGISTRY**
+[**--help**|**-h**]
+
+## DESCRIPTION
+**kpod login** logs into a specified registry server with the correct username
+and password. **kpod login** reads in the username and password from STDIN.
+The username and password can also be set using the **username** and **password** flags.
+The path of the authentication file can be specified by the user by setting the **outfile**
+flag. The default path used is **XDG\_RUNTIME\_DIR/containers/auth.json**.
+
+**kpod [GLOBAL OPTIONS]**
+
+**kpod login [GLOBAL OPTIONS]**
+
+**kpod login [OPTIONS] SERVER [GLOBAL OPTIONS]**
+
+## OPTIONS
+
+**--password, -p**
+Password for registry
+
+**--username, -u**
+Username for registry
+
+**--outfile, -o**
+Path of the authentication file. Default is XDG_\RUNTIME\_DIR/containers/auth.json
```

Comment:
I think you should add 
${XDG_\RUNTIME\_DIR}

Reply:
fixed